### DISCUSSION TITLE
能不能出个详细点教程

### DISCUSSION THREAD

**Discussion by nljie1103** [imsyy/DailyHotApi#22] [Q&A] [answered]
### 问题描述

小白不太搞得懂啊·

**Comment by nljie1103** [comment]
在自己服务器搭建的教程，vercel服务器太卡了

**Reply by imsyy** [reply]
#19

**Reply by dahaichaoyong** [reply, marked as answer]
宝塔部署DailyHotApi： https://github.com/imsyy/DailyHotApi/discussions/27

**Reply by James904141** [reply]
大佬按照这个方法pnpm install 后没出现dist文件夹

**Reply by James904141** [reply]
![QQ拼音截图20231212181213](https://github.com/imsyy/DailyHotApi/assets/145677216/70bb26d6-d1c4-4f49-88be-3e5c9bdaa5cc)
怎么办？
